pkg/project/admission/requestlimit/v1/types.go

```diff
@@ -0,0 +1,22 @@
+package v1
+
+import (
+	"k8s.io/kubernetes/pkg/api/unversioned"
+)
+
+// ProjectRequestLimitConfig is the configuration for the project request limit plug-in
+// It contains an ordered list of limits based on user label selectors. Selectors will
+// be checked in order and the first one that applies will be used as the limit.
+type ProjectRequestLimitConfig struct {
+	unversioned.TypeMeta
+	Limits []ProjectLimitBySelector `yaml:"limits",description:"project request limits"`
+}
+
+// ProjectLimitBySelector specifies the maximum number of projects allowed for a given user label selector
+type ProjectLimitBySelector struct {
+	// Selector is a user label selector. An empty selector selects everything.
+	Selector map[string]string `yaml:"selector",description:"user label selector"`
+	// MaxProjects is the number of projects allowed for this class of users. If MaxProjects is nil,
+	// there is no limit to the number of projects users can request.
+	MaxProjects *int `yaml:"maxProjects,omitempty",description:"maximum number of projects, unlimited if nil"`
```

Comment:
> Because you may want to have a global default limit (empty selector) and MaxProjects=1
> which is what I would think is the default for online.

Ok, that makes sense.  Can you add the reason why this is useful to the godoc at least?